workshops/templates/workshops/eventrequest_accept.html

```diff
@@ -0,0 +1,175 @@
+{% extends "workshops/_page_fluid_nosidebar.html" %}
+
+{% load crispy_forms_tags %}
+{% load selectable_tags %}
+
+{% block content-fluid %}
+<div class="row">
+  <div class="col-sm-6">
+    <h1>Workshop request #{{ object.pk }}</h1>
+    <table class="table table-striped">
+      <tbody>
+        <tr><th>Workshop type:</th>
+            <td>{{ object.get_workshop_type_display }}</td></tr>
+        <tr><th>Person:</th>
+            <td>{{ object.name }} &lt;{{ object.email|urlize }}&gt;</td></tr>
+        <tr><th>Affiliation:</th>
+            <td>{{ object.affiliation }}</td></tr>
+        <tr><th>Location:</th>
+            <td>{{ object.location }}</td></tr>
+        <tr><th>Country:</th>
+            <td>{{ object.country.name }} <img src="{{ object.country.flag }}" alt="{{ object.country }}" class="country-flag" /></td></tr>
```

Comment:
This actually works on Linux. Could you check if it works on Mac? I think I copied this from the package's sources, so they theoretically should have got it well.